src/components/layout/desktop_footer.tsx

```diff
@@ -0,0 +1,85 @@
+import * as React from "react"
+import styled from "styled-components"
+import * as fonts from "../../assets/fonts"
+
+const asideWidth = 28
+const columnWidth = (100 - asideWidth) / 4
+
+const Footer = styled.footer`
+  ${fonts.secondary.style}
+  background-color: white;
+  padding: 20px 0;
+  .column, .column-aside {
```

Comment:
I did inspect the DOM and it seemed alright to me.

The generated CSS is prefixed by the overall component one, so this did 👍 seem to work and not pollute too much.

<img width="385" alt="screen shot 2017-03-01 at 5 59 33 pm" src="https://cloud.githubusercontent.com/assets/1457859/23485269/bc9f2fa8-fea8-11e6-90c3-f6ba89dc3635.png">